examples/postgres/test_all

```diff
@@ -1,7 +1,22 @@
 #!/bin/sh
 set -e
 
-export DATABASE_URL="postgres://localhost/diesel_examples"
+set -a
+. ../../.env || true
+DATABASE_URL=${DATABASE_URL-${PG_DATABASE_URL}}
+set +a
+
+trap finish EXIT
+
+function finish {
+  # Make sure when we're done, we leave behind an empty database. Otherwise,
+  # at the time of writing, running diesel_tests stumbles as it tries to run
+  # conflicting migrations on an overlapping schema.
+  #
+  # TODO: Find a better way to avoid mixing of resources across tests.
+  ../../bin/diesel database drop
+  ../../bin/diesel database touch
```

Comment:
If there's something going on with our test suite that requires recreating the database every run, we absolutely need to fix that. This is not an acceptable solution.